tls/cipher_suites.go

```diff
@@ -1101,6 +1102,75 @@ var SafariNoDHECiphers []uint16 = []uint16{
 	TLS_RSA_WITH_RC4_128_MD5,
 }
 
+var PortableCiphers []uint16 = []uint16{
+	// stdlibCiphers, to preserve the default behavior for common cipher-suites that may be present
+	TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,
+	TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
+	TLS_ECDHE_RSA_WITH_RC4_128_SHA,
+	TLS_ECDHE_ECDSA_WITH_RC4_128_SHA,
+	TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
+	TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,
+	TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,
+	TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,
+	TLS_RSA_WITH_RC4_128_SHA,
+	TLS_RSA_WITH_AES_128_CBC_SHA,
+	TLS_RSA_WITH_AES_256_CBC_SHA,
+	TLS_ECDHE_RSA_WITH_3DES_EDE_CBC_SHA,
+	TLS_RSA_WITH_3DES_EDE_CBC_SHA,
+	// Most of the other implemented ciphers, in a somewhat reasonable order
+	TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA384,
+	TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,
+	TLS_ECDHE_ECDSA_WITH_3DES_EDE_CBC_SHA,
+	TLS_DHE_RSA_WITH_AES_128_GCM_SHA256,
+	TLS_RSA_WITH_AES_256_GCM_SHA384,
+	TLS_RSA_WITH_AES_128_GCM_SHA256,
+	TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA384,
+	TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256,
+	TLS_DHE_DSS_WITH_AES_128_CBC_SHA,
```

Comment:
I'm not sure that we actually can complete a handshake that uses `DSS` certs.

Reply:
@dadrian you are correct about DSS certs- so I will remove those (for starters) I still need to test the non-ephemeral cipher-suites you mentioned- and the ANON ones..  in the process of doing that now.

```
openssl.old s_server -dcert server2.pem -dkey server2.pem -cert server1.pem -key server1.pem  -cipher DH-DSS-AES256-GCM-SHA384:DHE-DSS-AES256-GCM-SHA384:DHE-DSS-AES256-SHA256:DH-DSS-AES256-SHA256:DHE-DSS-AES256-SHA:DH-DSS-AES256-SHA:DHE-DSS-CAMELLIA256-SHA256:DH-DSS-CAMELLIA256-SHA256:DHE-DSS-CAMELLIA256-SHA:DH-DSS-CAMELLIA256-SHA:SRP-DSS-AES-128-CBC-SHA:DH-DSS-AES128-GCM-SHA256:DHE-DSS-AES128-GCM-SHA256:DHE-DSS-AES128-SHA256:DH-DSS-AES128-SHA256:DHE-DSS-AES128-SHA:DH-DSS-AES128-SHA:DHE-DSS-CAMELLIA128-SHA256:DH-DSS-CAMELLIA128-SHA256:DHE-DSS-SEED-SHA:DH-DSS-SEED-SHA:DHE-DSS-CAMELLIA128-SHA:DH-DSS-CAMELLIA128-SHA:DHE-DSS-RC4-SHA:SRP-DSS-3DES-EDE-CBC-SHA:EDH-DSS-DES-CBC3-SHA:DH-DSS-DES-CBC3-SHA -tls1_2 -debug -msg -www
```
...

```
{"ip":"127.0.0.1","data":{"http":{"status":"unknown-error","protocol":"http","result":{},"timestamp":"2021-02-16T19:23:24-05:00","error":"tls: server's certificate contains an unsupported type of public key: *dsa.PublicKey"}}}
```